analysis/distance_comparison_routes_vs_people.py

```diff
@@ -0,0 +1,107 @@
+import pandas as pd
+from tqdm import tqdm
+from pdb import set_trace as st
+import os
+
+"""
+  Compares the distances of each person according to the output obtained from 0_people5to12.csv
+  and the sum of the edges obtained from 0_route5to12.csv and edges.csv.
+  Saves the merge of both distances for each person as a csv file.
+  In case of finding any discrepancies, outputs one of them.
+
+  Args:
+    edges_file: Path to the edges information file.
+
+    people_file: Path to the people information file. Default: "../LivingCity/0_people5to12.csv"
+
+    route_file: Path to the route information file. Default: "../LivingCity/0_route5to12.csv"
+
+    stop_if_discrepancy_found:  if true, stops as soon as it finds a discrepancy between
+                                the two distances of a person. Useful for testing without having to
+                                wait for the whole network to be processed. Default: true
+
+    output_file: Path to distance merge output. Default: distance_merge.csv
+                Outputted as csv with 3 columns: person_id,distance_sum_of_edges,distance_people_info
+  
+  Returns:
+    Nothing
+"""
+def merge_and_compare_route_vs_people_distances(
+      edges_file,
+      people_file = "../LivingCity/0_people5to12.csv",
+      route_file = "../LivingCity/0_route5to12.csv",
+      stop_if_discrepancy_found=True,
+      output_file="distance_merge.csv"):
+  
+  # check if output file already exists
+  if (os.path.isfile(output_file)):
+    answer = "no answer yet"
+    while(not answer.lower() in {"y","yes","n","no",""}):
+      answer = input("{} already exists. Do you want to replace it? (Y/n): ".format(output_file))
+    if (answer.lower() in {"n","no"}):
+      print("Stopping.")
+      return
+
+  # check if input files do not exist
+  for an_input_file in [edges_file,people_file,route_file]:
+    if (not os.path.isfile(an_input_file)):
+      raise FileNotFoundError("{} not found.".format(an_input_file))
+
+  print("Loading edges from {}...".format(edges_file))
+  print("Loading people from {}...".format(people_file))
+  print("Loading routes from {}...".format(route_file))
+  pd_people = pd.read_csv(people_file)
+  number_of_people = len(pd_people.index)
+  pd_edges = pd.read_csv(edges_file)
+
+  f = open(output_file, "w")
+  f.write("person_id,distance_sum_of_edges,distance_people_info\n")
+
+  print("Processing routes...")
+  # reads in chunks of 1000 to reduce memory usage
+  (discrepancy_person, discrepancy_distance_people_info, discrepancy_distance_sum_of_edges) = (None, None, None)
+  for chunk_route in tqdm(pd.read_csv(route_file, sep=":", chunksize=1000), total=number_of_people/1000):
```

Comment:
It would be great to define the chunk size in a constant along with other constants used in the code.